src/aluno/CadastroAlunoPage.tsx

```
@@ -84,28 +94,39 @@ export function CadastroAlunoPage() {
+
+      const response = await LoginService.login(loginRequest)
+      const responseData = response?.data
+      if (responseData?.token) {
+        handleLogin(responseData)
+      } else {
+        throw new Error('Token não recebido do servidor')
+      }
+      toast.success('Login realizado com sucesso!')
+      navigate('/dashboard')
+
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)